docs/field-values.asciidoc

```diff
@@ -331,6 +349,16 @@ The access event type is used for the subset of events within a category that in
 
 
 
+[float]
+[[ecs-event-type-admin]]
+==== admin
+
+The admin event type is used for the subset of events within a category that are related to admin objects. Common example: `event.category:iam AND event.type:admin`. You can further distinguish admin operations using the ECS `event.action` field.
```

Comment:
@MikePaquette Can you add a more specific example of what would map to "admin" to the description? What's an "admin object"?

Reply:
Something like administrative changes withingan identity and access management framework that do not specifically affect a user or group (adding new applications to federation solution for instance)?

Reply:
Added clarification based on Dain's examples.